CHANGELOG.md

```diff
@@ -1,5 +1,16 @@
 # eslint-config-digitalbazaar ChangeLog
 
+### 2.6.0 -
+
+### Added
+- Add rule `no-loss-of-precision`: 'error'.
+- Add rule `no-return-await`: 'error'.
+- Add tests for `no-loss-of-precision`.
+
+
+### Changed
+- Change peerDep for `eslint` to `7.8.0`.
```

Comment:
Looks like only the local dev dep was updated.  For this package try to be as forgiving as possible with the peer dep version.  Is 7.8.0 required for those new rules or can it be lower?  I'm guessing by now perhaps 5.13.x is too low?  Not sure if we should be doing bigger revisions with this package when the peer deps do major version changes.

Reply:
it can be lower, I just bumped it because `7.0.0` was the first eslint to support `es2020` and there have been A LOT of updates: https://github.com/eslint/eslint/releases

So I'm not opposed to using the lowest version possible, but I was thinking adding a peerDependency of the latest will remind people to update `eslint`.

@mattcollier opinions on if we should use a `peerDependency` of lowest as possible or latest as possible?